mobile/v1/redshift/sql-runner/sql/standard/01-base/XX-destroy/XX-destroy-base.sql

```diff
@@ -0,0 +1,18 @@
+/*
+   Copyright 2021 Snowplow Analytics Ltd. All rights reserved.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+DROP TABLE IF EXISTS {{.output_schema}}.{{.model}}_base_event_id_manifest{{.entropy}};
+DROP TABLE IF EXISTS {{.output_schema}}.{{.model}}_base_session_id_manifest{{.entropy}};
```

Comment:
Nit: Some files have no newline at the end, some do. Likely this is down to editor sessions - in the ideal world we have consistency across the project.

Reply:
Aligned to have newline across all files. God bless REGEX